lib/node_modules/@stdlib/array/bool/README.md

```diff
@@ -779,6 +779,106 @@ var count = context.count;
 
+<a name="method-reduce"></a>
+
+#### BooleanArray.prototype.reduce( reducerFn\[, initialValue] )
+
+Applies a provided callback function to each element of the array, in order, passing in the return value from the calculation on the preceding element and returning the accumulated result upon completion.
+
+```javascript
+var Boolean = require( '@stdlib/boolean/ctor' );
+
+function reducer( acc, v ) {
+    return Boolean( acc & v );
+}
+
+var arr = new BooleanArray( 3 );
+
+arr.set( true, 0 );
+arr.set( false, 1 );
+arr.set( true, 2 );
+
+var sum = arr.reduce( reducer );
+// returns false
+```
+
+The reducer function is provided four arguments:
+
+-   **acc**: accumulated result.
+-   **value**: current array element.
+-   **index**: current array element index.
+-   **arr**: the array on which this method was called.
+
+By default, the function initializes the accumulated result to the first element in the array and passes the second array element as `value` during the first invocation of the provided callback. To begin accumulation from a different starting value and pass in the first array element as `value` during the first invocation of the provided callback, provide an `initialValue` argument.
+
+```javascript
+function reducer( acc, v ) {
+    if ( v ) {
+        return acc + 1;
+    }
+    return acc;
+}
+
+var arr = new BooleanArray( 3 );
+
+arr.set( true, 0 );
+arr.set( false, 1 );
+arr.set( true, 2 );
+
+var sum = arr.reduce( reducer );
+// returns 2
+```
+
+<a name="method-reduce-right"></a>
+
+#### Complex64Array.prototype.reduceRight( reducerFn\[, initialValue] )
+
+Applies a provided callback function to each element of the array, in reverse order, passing in the return value from the calculation on the following element and returning the accumulated result upon completion.
+
+```javascript
+var Boolean = require( '@stdlib/boolean/ctor' );
+
+function reducer( acc, v ) {
+    return Boolean( acc & v );
+}
+
+var arr = new BooleanArray( 3 );
+
+arr.set( true, 0 );
+arr.set( false, 1 );
+arr.set( true, 2 );
+
+var sum = arr.reduceRight( reducer );
+// returns false
+```
+
+The reducer function is provided four arguments:
+
+-   **acc**: accumulated result.
+-   **value**: current array element.
+-   **index**: current array element index.
+-   **arr**: the array on which this method was called.
+
+By default, the function initializes the accumulated result to the last element in the array and passes the second-last array element as `value` during the first invocation of the provided callback. To begin accumulation from a different starting value and pass in the last array element as `value` during the first invocation of the provided callback, provide an `initialValue` argument.
+
+```javascript
+function reducer( acc, v ) {
+    if ( v ) {
+        return acc + 1;
+    }
+    return acc;
+}
+
+var arr = new BooleanArray( 3 );
+
+arr.set( true, 0 );
+arr.set( false, 1 );
+arr.set( true, 2 );
+
+var sum = arr.reduceRight( reducer );
```

Comment:
```suggestion
var sum = arr.reduceRight( reducer, 0 );
```

Missing the initial value, which is what this example is supposed to illustrate.